config/database.yml

```diff
@@ -58,7 +58,8 @@ development:
 test:
   <<: *default
   database: Rails_Blog_App_test
-
+  username: "Omar Muhammad"
+  password: 1234
```

Comment:
- [ ]  [OPTIONAL] I suggest the use of environment variables to store sensitive data. For example you can make use of the rails inbuilt credentials - <%= Rails.application.credentials.username %> then store the values in the encrypted credentials file. To open it use rails credentials:edit in the terminal